proxysql-admin

```diff
@@ -1110,12 +1112,12 @@ function enable_proxysql() {
     proxysql_load_to_runtime_save_to_disk "MYSQL SERVERS" $LINENO
```

Comment:
not sure if `MODE=loadbal` possible in native Galera Clustering

Reply:
@delgod loadbal mode will fail with the following error.
```
Application user 'cluster_two'@'127.%' already present in PXC.

Adding the Percona XtraDB Cluster server nodes to ProxySQL
ERROR 1045 (28000) at line 1: CHECK constraint failed: mysql_galera_hostgroups
ERROR (line:1238) : Failed to add the PXC monitoring scheduler in ProxySQL.
-- Please check the ProxySQL connection parameters and status.
ramesh@qaserver-05:/qa/8pxc/work/proxysql-admin-tool$
```

Reply:
The loadbal mode is essentially (1) all nodes are writers and (2) max writers is a large number (say 10000).